deployment.py

```diff
@@ -139,7 +131,31 @@ def __init__(self):
                     {
                         "flow": "sym",
                         "proto": "all",
-                        "groups": ("internal", "web")
+                        "groups": ("internal", "temp", "web")
+                    },
+                )
+            },
+
+            {
+                "name": "temp",
+                "rules": (
+                    {
+                        "flow": "in",
+                        "proto": "tcp",
+                        "port": [22, 80, 443, 8080],
+                        "cidr_ip": "0.0.0.0/0"
+                    },
+
+                    {
+                        "flow": "out",
+                        "proto": "all",
+                        "cidr_ip": "0.0.0.0/0"
+                    },
```

Comment:
I'm no expert on AWS deployment, but I think you could tighten the outbound restrictions on the web and temp groups.  The web group should only need tcp out in the ephemeral range of ports, unless it, in turn, is making http requests somewhere.  The temp group should also be able to be limited to tcp out, though I might leave the entire range of ports open since I assume various provisioning occurs while this security is in effect.